### PR TITLE
Unit test frame

### PR DESCRIPTION
**Pull request by SharpRazor**
I have completed a simple UT for this project but still need to fully compete some cases like test_run_instances ,test_terminate_instance and all the cases under class VOLUMEOps.
all other cases passed the test:
$ python test_zvmsdk.py 
Ran 27 tests in 0.013s
OK

### CODE REVIEW

File: tests/test_zvmsdk.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+        self.volumeops = sdkapi._get_volumeops()
+        '''
+        return_value = {
+                'os':'rhel7',
+                'vcpus':1,
+                'ip_addr':'192.168.114.6',
+                'power_state':'off',
+                'memory':'1024m',
+                }
+        '''
```
Comment (bjhuangr) [inline tests/test_zvmsdk.py]: setUp() has return value?

File: tests/test_zvmsdk.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+    def tearDown(self):
+        pass
```
Comment (bjhuangr) [inline tests/test_zvmsdk.py]: Do you really want to overwrite tearDown() here?

File: tests/test_zvmsdk.py [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+        xrequest.assert_called_with('PUT', url, body)
+        delete_info.assert_called_with(uuid)
+
+    # TODO
+    def test_attach_volume(self):
+        self.assertFalse('foo'.isupper())
+
+    # TODO
+    def test_detach_volume(self):
+        self.assertFalse('foo'.isupper())
```
Comment (bjhuangr) [inline tests/test_zvmsdk.py]: leave a "pass" in this case is fine

File: tests/test_zvmsdk.py (hunk 1 above)
Comment (SharpRazor) [inline tests/test_zvmsdk.py]: I just add a comment here for possible future use...

File: tests/test_zvmsdk.py (hunk 2 above)
Comment (SharpRazor) [inline tests/test_zvmsdk.py]: No , I have not determined how to deal with tearDown() yet.

File: tests/test_zvmsdk.py (hunk 3 above)
Comment (SharpRazor) [inline tests/test_zvmsdk.py]: OK

**Pull request merged**